demos/main/src/main/java/com/google/android/exoplayer2/demo/upstream/EntityUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.google.android.exoplayer2.demo.upstream;
+
+import com.google.android.exoplayer2.demo.upstream.entity.ProtocolData;
+import com.google.android.exoplayer2.demo.upstream.entity.StreamResponse;
+
+public class EntityUtil {
+
+  private EntityUtil() {
+  }
+
+  // Util methods.
+
+  public static ProtocolData getProtocolData(boolean forceDash, StreamResponse result) {
```

Comment:
Maybe add these 3 static methods to `ProtocolData` itself? then we can get rid of this class

Reply:
done